packages/cid-verifier/test/verification.spec.js

```diff
@@ -1,17 +1,24 @@
+// @ts-ignore
+import Hash from 'ipfs-only-hash'
 import { test, getMiniflare } from './utils/setup.js'
 import { toDenyListAnchor } from '../src/utils/denylist.js'
 
+/**
+ * @param {string} s
+ */
+const createTestCid = async (s) => (new Buffer(await Hash.of(s))).toString('base64')
```

Comment:
So, this should be `await Hash.of(s)` only right?

Reply:
Let's use CIDv1 though, I think we can do `await Hash.of(s, { cidVersion: 1 })` to have a valid cid string of version 1